quarry/web/worker.py

```diff
@@ -148,6 +148,13 @@ def run_query(query_run_id):
             celery_log.info("Stopped run for qrun:%s", qrun.id)
         else:
             write_error(qrun, e.args[1])
+    except pymysql.err.OperationalError as e:
```

Comment:
I'm not sure if this would work. I believe the `pymysql.err.OperationalError` raised is being wrapped by an `sqlalchemy.exc.OperationalError`. We could just catch `Exception`, so that everything is covered.

Reply:
Thanks, changed to catching all Exceptions